wagtail/core/blocks/field_block.py

```diff
@@ -22,6 +22,44 @@ class FieldBlock(Block):
     def id_for_label(self, prefix):
         return self.field.widget.id_for_label(prefix)
 
+    def prepare_value(self, value, errors=None):
+        from wagtail.admin.rich_text import DraftailRichTextArea
+        from wagtail.admin.widgets import AdminDateInput, AdminDateTimeInput
+
+        value = self.value_for_form(self.field.prepare_value(value))
+        widget = self.field.widget
+        if isinstance(self, RichTextBlock) \
+                and isinstance(widget, DraftailRichTextArea):
+            value = widget.format_value(value)
+        if isinstance(widget, (AdminDateInput, AdminDateTimeInput)):
+            value = widget.format_value(value)
+        return value
+
+    def prepare_for_react(self, parent_block, value,
+                          type_name=None, errors=None):
+        data = super().prepare_for_react(parent_block, value,
+                                         type_name=type_name, errors=errors)
+        if errors:
+            data['html'] = self.render_form(
+                value, prefix=Block.FIELD_NAME_TEMPLATE, errors=errors)
+        return data
+
+    def get_definition(self):
+        definition = super(FieldBlock, self).get_definition()
```

Comment:
Since Wagtail does not support Python 2.x, there is no need to pass parameters to `super` (it's not the only occurrence). Old habits die hard 😄

Reply:
Aha, indeed, I forgot to replace it when making the PR from the monkey-patched application (that needs it because of monkey patching, not because of Python 2).